tests/bytes.rs

```diff
@@ -858,10 +889,10 @@ fn script_input_coin_data_offset() {
                         );
 
                         let (offset, len) = tx
-                            .input_coin_predicate_offset(offset)
+                            .input_predicate_offset(offset)
                             .expect("Failed to fetch offset");
 
-                        assert_ne!(predicate.len(), len);
+                        assert_ne!(bytes::padded_len(&predicate), predicate.len());
```

Comment:
Do we need to test message predicate offsets for scripts too? Looks like it was only added for the create tx offset test.

Reply:
The offset is not bound to the transaction variant. We can ofc test that as sanity check, but in practice it won't increase the robustness of the tests